main/main.cpp

```diff
@@ -1406,6 +1406,8 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 
 	InputDefault *id = Object::cast_to<InputDefault>(Input::get_singleton());
 	if (id) {
+		agile_input_event_flushing = GLOBAL_DEF("input_devices/buffering/agile_event_flushing", false);
+
```

Comment:
Shouldn't this property be used to update the `InputDefault` singleton using `InputDefault::set_use_input_buffering(...)`?
If not, what's the intended use for the method?